### PR TITLE
Initial role PR

### PR DESCRIPTION
**Pull request by haslersn**
* [x] Add `README.md`
* [x] Add `meta/main.yml`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# Ansible role for the BIRD Internet Routing Daemon
+
+This role installs and configures
+[the BIRD Internet Routing Daemon](https://bird.network.cz).
```
Comment (SlothOfAnarchy) [inline README.md]:
```suggestion
[the BIRD Internet Routing Daemon](https://bird.network.cz) on an APT-based system.
```

File: tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- name: Configure bird6 service
+  service:
+    name: bird6
+    enabled: "{{ bird_ipv6_config != [] }}"
+    state: "{% if bird_ipv6_config != [] %}started{% else %}stopped{% endif %}"
+
+- meta: flush_handlers
+
+- name: Enable bird service
+  service:
```
Comment (SlothOfAnarchy) [inline tasks/main.yml]: I think you're missing a condition (that `bird_ipv6_config` is not empty). `bird6` is missing, too.

File: templates/bird.conf.j2 [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{% macro print_config(config) %}
+{% if config is string %}
+{{ config }}
+{%- elif config is mapping %}
+{% for key, value in config.items() %}
+{{ key }} {
+    {{ print_config(value) | indent }}}
+{%- endfor %}{# for key, value in element.items() #}
```
Comment (SlothOfAnarchy) [inline templates/bird.conf.j2]: Why this commented-out `for` header?

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: templates/bird.conf.j2 (hunk 3 above)
Comment (haslersn) [reply templates/bird.conf.j2]: Such that you know which loop this `endfor` belongs to.

File: tasks/main.yml (hunk 2 above)
Comment (haslersn) [reply tasks/main.yml]: Thanks. This is now fixed.

File: templates/bird.conf.j2 (hunk 3 above)
Comment (SlothOfAnarchy) [reply templates/bird.conf.j2]: In that case:
```suggestion
{%- endfor %}{# for key, value in config.items() #}
```

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

**Review by SlothOfAnarchy — DISMISSED** [review]

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**